### DISCUSSION TITLE
MouseInc建议

### DISCUSSION THREAD

**Discussion by cc19264997** [shuax/MouseInc.Settings#264] [General]
![QQ图片20220415225009](https://user-images.githubusercontent.com/25716656/163585856-9ac5518f-d7e4-47bc-958d-4d824478ebf0.png)

能否让边缘滚动翻页的位置往下面一点点呢。

**Comment by shuax** [comment]
建议使用专门的浏览器增强工具

**Reply by cc19264997** [reply]
试过，可是您的FireDoge对95以上的版本失效了
